driver-core/src/main/java/com/datastax/driver/core/TypeCodec.java

```diff
@@ -533,8 +535,8 @@ public DataType getCqlType() {
      * {@link TupleValue}, and the internal representation of a {@code ROWS} response),
      * which may appear in driver logs.</li>
      * </ol>
-     * If choose not to implement this method, you can return a constant string (for example
-     * "XxxCodec.format not implemented").
+     * If you choose not to implement this method, you should not throw an exception but
+     * instead return a constant string (for example "XxxCodec.format not implemented").
```

Comment:
Mmm after re-reading this javadoc I remember the rationale for returning a string and never throwing an exception. The real issue is that `MappedUdtCodec` was throwing an `UnsupportedOperationException`, that was an oversight.
Provided that all our internal codecs properly implement `format`, We don't even need the catch block in `BuiltStatement.toString()`, although it doesn't hurt to keep it there.